docs/backend-api/resources/tracking/tracker/sensor/index.md

```diff
@@ -52,6 +55,9 @@ Contains API calls to interact with sensors.
     * `max` - optional. Double. Max acceptable raw value for a sensor.
     * `max_lowering_by_time` - optional. Double. Max legal value lowering per hour.
     * `max_lowering_by_mileage` - optional. Double. Max legal value lowering per 100 km.
+    * `ignore_drains_in_move` - optional. Boolean. Default is false. If true, fuel level decreasing detected during movement will not be marked as fuel drain.
```

Comment:
"fuel level decreasing" - it's a normal situation. We should underline abrupt decreasing or increasing. I suggest another wording: "If `true`, the fuel drains will not de detecting during movement."